oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/query/IndexName.java

```diff
@@ -237,9 +242,44 @@ private static boolean isIndexActive(String indexPath, NodeState rootState) {
                 return true;
             }
         }
+        if (isIndexActiveMerged(indexNode, rootState)) {
+            return true;
+        }
         return false;
```

Comment:
minor: this could be simplified with
```suggestion
        return isIndexActiveMerged(indexNode, rootState);
```